### DISCUSSION TITLE
Use of Lightroom 5.7.1

### DISCUSSION THREAD

**Discussion by nickbrocious** [rsjaffe/MIDI2LR#724] [Q&A]
I was very generously gifted a Behringer X-touch Mini MIDI Controller and quickly found this community, but the problem I've run into is the fact that I am using Lightroom 5.7.1 and am unable to afford to upgrade via Adobe's Creative Cloud. Is there any version of this that would work with LR 5.7.1? Or am I SOL without a newer version of LR? Maybe there's a different plugin I could download? I'd really love some help.

**Comment by antzpantz** [comment]
Apparently try "version 3 for earlier versions of Lightroom" as per the
MIDI2LR main page. The latest version 3 is 3.5.0.3:

https://github.com/rsjaffe/MIDI2LR/releases/tag/v3.5.0.3

Give it a try and see  if it works for you.

**Reply by nickbrocious** [reply]
This doesn't work as the version you've referenced, as well as all other previous versions, are for "Lightroom 6+"

**Reply by fyrus7** [reply]
> Apparently try "version 3 for earlier versions of Lightroom" as per the MIDI2LR main page. The latest version 3 is 3.5.0.3: https://github.com/rsjaffe/MIDI2LR/releases/tag/v3.5.0.3 Give it a try and see if it works for you.
> […](#)
> On Sat, 31 Dec 2022 at 03:29, nickbrocious ***@***.***> wrote: I was very generously gifted a Behringer X-touch Mini MIDI Controller and quickly found this community, but the problem I've run into is the fact that I am using Lightroom 5.7.1 and am unable to afford to upgrade via Adobe's Creative Cloud. Is there any version of this that would work with LR 5.7.1? Or am I SOL without a newer version of LR? Maybe there's a different plugin I could download? I'd really love some help. — Reply to this email directly, view it on GitHub <#724>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AFIYPFF7BIKWPQYFDWS4FBTWP4Z3BANCNFSM6AAAAAATNDHQTI> . You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

v3.5.0.3 only work with LR9 as i try with LR3 & LR5 its not working until i use LR9
